### ISSUE TITLE
Can't open project with Xcode 12.5.1

### ISSUE THREAD

**Issue by DevulderJeanPaul** [dagronf/DSFBrowserView#1]
Error are
The project at '/Users/rgb/Downloads/DSFBrowserView-main/Demos/DSFBrowserView Demo/DSFBrowserView Demo.xcodeproj' cannot be opened because it is in a future Xcode project file format. Adjust the project format using a compatible version of Xcode to allow it to be opened by this version of Xcode.

Project using Xcode 13 ?

**Comment by dagronf** [comment]
Hey mate,

You're absolutely right - I've must have created this demo project using Xcode 13 beta.  If you grab the 0.5.4 tag (or main) I've downgraded the projects so that they can be opened, compiled and run on Xcode 12.5

Would be great to hear if you're successful with the new version -- Xcode 13's 'set compatibility' by default builds apps that, when run, have absolutely nothing in them!  Had to do some tweaking to get it to work but once I figured out what it was doing the projects seem to be fine on both Xcode 12 and Xcode 13 now.

Many thanks for the heads up!
